odoo/addons/base/rng/calendar_view.rng

```diff
@@ -20,6 +20,8 @@
             <rng:optional><rng:attribute name="color" /></rng:optional>
             <rng:optional><rng:attribute name="event_open_popup" /></rng:optional>
             <rng:optional><rng:attribute name="js_class"/></rng:optional>
+            <rng:optional><rng:attribute name="hide_time"/></rng:optional>
+            <rng:optional><rng:attribute name="hide_date"/></rng:optional>
```

Comment:
Note that the doc on calendar should also be adapted to mention these two new attributes.